msg/sensor_correction.msg

```diff
@@ -6,52 +6,21 @@ uint64 timestamp		# time since system start (microseconds)
 
 # Corrections for gyro angular rate outputs where corrected_rate = raw_rate * gyro_scale + gyro_offset
 # Note the corrections are in the sensor frame and must be applied before the sensor data is rotated into body frame
-# corrections for uORB instance 0
-float32[3] gyro_offset_0	# gyro XYZ offsets in the sensor frame in rad/s
-float32[3] gyro_scale_0		# gyro XYZ scale factors in the sensor frame
-
-# corrections for uORB instance 1
-float32[3] gyro_offset_1	# gyro XYZ offsets in the sensor frame in rad/s
-float32[3] gyro_scale_1		# gyro XYZ scale factors in the sensor frame
-
-# corrections for uORB instance 2
-float32[3] gyro_offset_2	# gyro XYZ offsets in the sensor frame in rad/s
-float32[3] gyro_scale_2		# gyro XYZ scale factors in the sensor frame
+uint32[3] gyro_device_ids
+float32[3] gyro_offset_0	# gyro 0 XYZ offsets in the sensor frame in rad/s
+float32[3] gyro_offset_1	# gyro 1 XYZ offsets in the sensor frame in rad/s
+float32[3] gyro_offset_2	# gyro 2 XYZ offsets in the sensor frame in rad/s
 
 # Corrections for acceleromter acceleration outputs where corrected_accel = raw_accel * accel_scale + accel_offset
 # Note the corrections are in the sensor frame and must be applied before the sensor data is rotated into body frame
-# corrections for uORB instance 0
-float32[3] accel_offset_0	# accelerometer XYZ offsets in the sensor frame in m/s/s
-float32[3] accel_scale_0	# accelerometer XYZ scale factors in the sensor frame
-
-# corrections for uORB instance 1
-float32[3] accel_offset_1	# accelerometer XYZ offsets in the sensor frame in m/s/s
-float32[3] accel_scale_1	# accelerometer XYZ scale factors in the sensor frame
-
-# corrections for uORB instance 2
-float32[3] accel_offset_2	# accelerometer XYZ offsets in the sensor frame in m/s/s
-float32[3] accel_scale_2	# accelerometer XYZ scale factors in the sensor frame
+uint32[3] accel_device_ids
+float32[3] accel_offset_0	# accelerometer 0 XYZ offsets in the sensor frame in m/s/s
+float32[3] accel_offset_1	# accelerometer 1 XYZ offsets in the sensor frame in m/s/s
+float32[3] accel_offset_2	# accelerometer 2 XYZ offsets in the sensor frame in m/s/s
```

Comment:
And no more accel scale calibration?

Reply:
Currently in master if you have TC enabled the regular calibration is storing it to TC parameters. The scale factor is stored directly (eg CAL_ACC0_XSCALE ->  TC_A0_SCL_0). The offset (eg CAL_ACCx_XOFF) is applied by adjusting the TC coefficients  (x0, y0, z0 terms).

The new idea is to basically keep them as separate as possible. Regular calibration (CAL_ACC*) is always there and operates the same way (on top of TC adjustment if available). Calibration code doesn't need to poke into TC parameters at all. This is important because in some cases the TC parameters are stored read only.